### DISCUSSION TITLE
Latest and Greatest: OS v0.1.7, Website and Ez Updates

### DISCUSSION THREAD

**Discussion by ntorga** [goinfinite/os#200] [Announcements]
Hello there, everyone,

We’d like to share this month’s latest updates on Infinite OS and Infinite Ez.

### - Website
We’ve finally got a webpage for Infinite [OS](https://goinfinite.net/os/) and [Ez](https://goinfinite.net/ez/) up and running! There, you’ll find the highlights of each project along with some screenshots, so you can get to know the projects without having to run them. In fact, the online demo is back and fully functional!

### - First Account Through the Browser
Infinite OS no longer requires you to dive into the terminal to create an account via CLI. In this latest version, once you access the dashboard at [https://localhost:1618](https://localhost:1618), you’ll be able to create your first account directly from the browser.

![setup-wizard](https://github.com/user-attachments/assets/dd7c9104-0fb7-46ac-874e-8d908e92402f)

So now, to deploy OS, you only need the deploy command, and that’s it:

```
docker run --rm --name myapp-container \
  --env 'PRIMARY_VHOST=myapp.net' \
  -p 8080:80 -p 8443:443 -p 1618:1618 \
  -it docker.io/goinfinite/os:latest
```

### - Cron Page Refactor
The Crons page has been refactored with HTMX+Alpine.js. It’s a transparent change for users, but a significant milestone for the developers.

![crons](https://github.com/user-attachments/assets/9fcae9fb-e6f3-4f88-b883-0ace66417a16)

### - Marketplace Deploy Modal
Now, the install parameters section is open by default, and the required fields, such as Admin Username or Admin Password, will be auto-generated. We highly recommend you change the default values, as they’re just there to give you an example of what we expect you to fill in.

![install-wp-marketplace](https://github.com/user-attachments/assets/2dd1263d-3092-48b2-a16b-267f33134bd9)

As always, we’ve fixed some minor bugs, including:

- Marketplace and Services repository is now branch-separated, maintaining retro-compatibility as the manifests are updated to include new system fields from new OS versions;
- Marketplace install parameters were being incorrectly parsed by the API;
- Marketplace and Services assets have been migrated to their own repository instead of relying on the goinfinite.net website;
- NGINX is now auto-restarted when a new SSL is issued;

## [Infinite Ez](https://github.com/goinfinite/ez)
There isn’t much news on the Ez side, as we’re currently working on the Backups feature. It’s a massive feature that will likely take a month or two to be released due to its complexity.

Since the last release note, there were just three bugs included in the re-released 0.0.9 version:

```
fix: account resource usage bug when no containers
fix: improve sys-install verbosity
fix: broken unocss css reset
```
---

That’s all for now, everyone. Keep an eye on our updated roadmap for [OS](https://github.com/orgs/goinfinite/projects/9/views/1) and [Ez](https://github.com/orgs/goinfinite/projects/8/views/1), and we'll catch up with you in the next update.

Thank you for your continued support on this journey. Cheers!

**Comment by felipoantonoff** [comment]
Muito bom, ficou bem interessante os novos recursos, sites e screenshots, ansioso para novas versões. 
Legal que na tela do Marketplace tem o E-commerce Opencart.
